configure.ac

```diff
@@ -306,6 +306,9 @@ AC_ARG_WITH(group-name-max-length,
 AC_ARG_WITH(su,
 	[AC_HELP_STRING([--with-su], [build and install su program and man page @<:@default=yes@:>@])],
 	[with_su=$withval], [with_su=yes])
+AC_ARG_WITH(subids-helpers,
+	[AC_HELP_STRING([--with-subids-helpers], [build and install subordinate ids helpers and man pages @<:@default=no@:>@])],
+	[with_subids_helpers=$withval], [with_subids_helpers=no])
```

Comment:
Do we really need a new `configure` flag?

I'd like to see the `getsubids` binary to be always installed when `newuidmap`  / `newgidmap` is installed.

Reply:
I added the flag just in case some distribution doesn't want to include the binary, but I wouldn't mind removing it as it would reduce the review and maintenance effort.

Reply:
Up to you @ikerexxe .  It's not setuid so I don't mind it always being included.  Please shout here today if you want me to wait for an update to remove it, else I'll go ahead and merge.

Reply:
Then, let's include it always. Let me change that today.